packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -137,11 +137,23 @@ export const StyledBaseInput = React.forwardRef<
       numberOfLines,
       isTextArea,
       hasPopup,
+      shouldIgnoreBlurAnimation,
+      setShouldIgnoreBlurAnimation,
       ...props
     },
     ref,
   ) => {
     const buttonValue = props.value ? props.value : props.defaultValue;
+    const commonProps = {
+      onBlur: (): void => {
+        if (shouldIgnoreBlurAnimation && setShouldIgnoreBlurAnimation) {
+          setShouldIgnoreBlurAnimation(false);
+          return;
+        }
+        setCurrentInteraction('default');
+      },
+      isFocused: currentInteraction === 'active',
```

Comment:
Suggestion: add some comments to explain the "why"

Reply:
done